packages/aws-cdk-lib/cx-api/lib/features.ts

```diff
@@ -1371,6 +1372,24 @@ export const FLAGS: Record<string, FlagInfo> = {
     recommendedValue: true,
     compatibilityWithOldBehaviorMd: 'Disable the feature flag to allow unsecure OIDC connection.',
   },
+
+  //////////////////////////////////////////////////////////////////////
+  [S3_BUCKET_DEFAULT_BLOCK_PUBLIC_ACCESS_PROPERTIES_TO_TRUE]: {
+    type: FlagType.BugFix,
+    summary: 'When enabled, the properties of class BlockPublicAccess will default to true',
+    detailsMd: `
+      Without this flag, the 'blockPublicAccess' property has a counter-intuitive and inconsistent behavior.
+      When the property value is not specified, then all the 4 member properties (blockPublicAcls,
+      ignorePublicAcls, blockPublicPolicy and restrictPublicBuckets) will default to 'true'. However in
+      cases where selected properties are explicitly set to false, the remaining properties for which no value
+      was specified will also default to 'false'.
+      
+      Intuitively, if the property is not set explicitly, it must default to 'true'. Enabling this flag will exhibit
```

Comment:
```suggestion
      Without this flag, the \`blockPublicAccess\` property has a counter-intuitive and inconsistent behavior.
      When the property value is not specified, then all the 4 member properties (\`blockPublicAcls\`,
      \`ignorePublicAcls\`, \`blockPublicPolicy\` and \`restrictPublicBuckets\`) will default to \`true\`. However, in
      cases where selected properties are explicitly set to \`false\`, the remaining properties for which no value
      was specified will also default to \`false\`.
      
      Intuitively, if the property is not set explicitly, it must default to \`true\`. Enabling this flag will exhibit
```
Escaped backticks should be used for prop names and values.